core/client/pom.xml

```diff
@@ -226,8 +226,34 @@
         </dependency>
       </dependencies>
     </profile>
-  </profiles>
 
+    <!-- build profile for Facebook Presto -->
+    <!-- Profile used to build alluxio client jar used for Facebook Presto. This will be activated by using -Ppresto-->
+    <!-- in the maven command line. -->
+    <profile>
+      <id>presto</id>
+      <!-- The dependencies are the same as the default dependencies, except that curator-client, curator-framework, -->
```

Comment:
could you update this comment? we are excluding guava, jackson-databind and jackson-core
btw, is excluding jackson-databind and jackson-core required?